source/localizable/feedspec/index.el.html.md.erb

```diff
@@ -628,6 +630,236 @@ To χρώμα του προϊόντος. Το πεδίο χρώματος λει
+##### Παράδειγμα
+
+~~~ xml
+<availability>1 to 3 days</availability>
+~~~
+
+
+#### Variation Size
+
+Το πεδίο αυτό πρέπει να αναφέρει το μέγεθος για ένα συγκεκριμένο size variation.
+
+Π.χ. ένα αθλητικό παπούτσι που υπάρχει στo μεγέθος 38 χρειάζεται να
+έχει στο XML την παρακάτω εγγραφή:
+
+~~~ xml
+<size>38</size>
+~~~
+
+Τα μισά μπορούν να δηλώνονται και ως  `1/2`, π.χ.
+
+~~~ xml
+<size>38 1/2</size>
+~~~
+
+Επίσης σε περίπτωση ρούχων τα μεγέθη μπορεί να έχουν ένα από τα παρακάτω format:
+
+~~~ xml
+<size>XL</size>
+~~~
+
+~~~ xml
+<size>Extra Large</size>
+~~~
+
+~~~ xml
+<size>10</size>
+~~~
+
+Αν ένα size variation έχει αναγνωριστικό για ένα μέγεθος διαστημα (π.χ. κάλτσες), τότε
+μπορείτε να το δηλώσετε ως *range* χρησιμοποιώντας την παύλα (-). Παράδειγμα:
+
+~~~ xml
+<size>5.5-6.5</size>
+~~~
+
+Τέλος, αν ένα size variation έχει παραπάνω από ένα αναγνωριστικά για ένα μεγέθος (π.χ. παντελόνια) τότε μπορούν
+να δηλωθούν και τα δύο με την χρήση της καθέτου (/). Παράδειγμα:
+
+~~~ xml
+<size>5.5/42</size>
+~~~
+
+> Size variations όπου το μέγεθος είναι απαραίτητο και δεν το περιλαμβάνουν (όπως π.χ.
+υποδήματα, πουκάμισα, κτλ) **δεν** θα ταξινομούνται.
+
+
+#### Variation Quantity
+
+Στο πεδίο αυτό μπορείτε να συμπληρώσετε την ποσότητα του size variation που έχετε διαθέσιμη
+για πώληση μέσω του Skroutz.
+
+> Το πεδίο δέχεται ακέραιες τιμές μεγαλύτερες ή ίσες του `0` και μικρότερες του `10000000`.
+
+> Αν το size variation δεν είναι διαθέσιμο, τότε η ενδεδειγμένη τιμή είναι `0`.
+
+##### Παράδειγμα
+
+~~~ xml
+<quantity>10</quantity>
+~~~
+
+
+#### Variation Link
+
+Είναι ο σύνδεσμος που οδηγεί στη σελίδα του καταστήματός σας για το συγκεκριμένο
+size variation. Θα πρέπει να είναι έγκυρος σύνδεσμος και να ξεκινάει με `https`.
+
+> Τα links **ΔΕΝ** πρέπει να είναι URL-encoded.
+
+> O σύνδεσμος αυτός είναι απαραίτητο να οδηγεί σε σελίδα προϊόντος και όχι σε
+κατηγορία ή συλλογή προϊόντων.
+
+##### Παράδειγμα
+
+~~~ xml
+<link>https://www.mydomain.com/products/1</link>
+<url>https://www.mydomain.com/products/product.php?itemid=1</url>
+<url><![CDATA[https://www.mydomain.com/products/product.php?itemid=1&amp;somevar=3]]></url>
+<url>https://www.mydomain.com/products/product.php?itemid=1&amp;amp;somevar=3</url>
+~~~
+
+
+#### Variation Price
+
+Είναι η τιμή του size variation **με Φ.Π.Α.**.
+
+> Η τιμή θα πρέπει να είναι ίδια για όλους και να μην έχει προϋποθέσεις
+εφαρμογής.
+
+> Το πεδίο τιμής μπορεί να δεχτεί μέχρι 2 δεκαδικά ψηφία.
+
+##### Παράδειγμα
+
+~~~ xml
+<price>23.22</price>
+<pricevat>23.22</pricevat>
+~~~
+
+
+#### Variation MPN
+
+Είναι ο μοναδικός κωδικός που δίνει ο κατασκευαστής στα προϊόντα του.
+Χρησιμοποιείται για την ταυτοποίηση των προϊόντων σε περίπτωση που αυτό δεν
+είναι εμφανές από το μοντέλο.
+
+> Το πεδίο αυτό είναι υποχρεωτικό για τις κατηγορίες όπου ο κωδικός του
+κατασκευαστή είναι σημαντικός για την αναγνώριση και ταξινόμηση του προϊόντος
+(π.χ. Τηλεοράσεις, Κινητά τηλέφωνα, Ρολόγια Χειρός, Αθλητικά Παπούτσια κτλ)
```

Comment:
Maybe we should remove this. It is only applicable in products and not variations.